joomla/joomla-cms
# [5.2] Media manager fix default adapter handling

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/44277 .

### Summary of Changes

Media manager produces  an error when "Path to Files Folder" field contain folder that not in the plugin "FileSystem - Local".
I have fixed fallback to first available adapter.


### Testing Instructions

Edit Media manager global options, set "Path to Files Folder" to some random existing folder.
Make sure this folder does not selected in plugin "FileSystem - Local".
Enable Drag&Drop upload for TinyMCE.
Open Article editing.



### Actual result BEFORE applying this Pull Request
An error page


### Expected result AFTER applying this Pull Request
No error


### Link to documentations
Please select:
